Code/Tools/AssetProcessor/AssetBuilderSDK/AssetBuilderSDK/AssetBuilderSDK.h

```diff
@@ -144,15 +144,16 @@ namespace AssetBuilderSDK
     {
         Platform_NONE       = 0x00,
         Platform_PC         = 0x01,
-        Platform_ANDROID    = 0x02,
-        Platform_IOS        = 0x04,
-        Platform_MAC        = 0x08,
+        Platform_LINUX      = 0x02,
```

Comment:
is the "enum Platform" ever written out to disk? if so, the Platform_LINUX should be added to the end to not renumber serialized data... 

otherwise no problem here, move along...

Reply:
This information is not serialized out.
I don't think this code is even compiled in the first place.

Reply:
Code\Tools\AssetProcessor\AssetBuilderSDK\AssetBuilderSDK\AssetBuilderSDK.cpp seems to include the H file and use the values

Reply:
@lumberyard-employee-dm It looks like that define that it is checking is [unconditionally enabled](https://github.com/o3de/o3de/blob/16ea6355a761fa74add9a46fd349bee735d1d329/Code/Tools/AssetProcessor/AssetBuilderSDK/AssetBuilderSDK/AssetBuilderSDK.h#L25-L32).

But I don't see anywhere that reflects this enum to the serialize context, so I think we're good.

Reply:
It doesn't seem legacy, if it is always on.
Well it is fine to keep on then